pages/school-administrators.md

```diff
@@ -2,11 +2,11 @@
 layout: page-breadcrumbs.html
 template: education-sco
 title: Resources for schools
-display_title: School administrators
+display_title: Resources for schools
 heading:
 permalink:
 source: https://www.benefits.va.gov/gibill/resources/education_resources/school_certifying_officials/sco_info.asp
-lastupdate: 2023-11-20
+lastupdate: 2023-12-22
```

Comment:
I take it this date comes from somewhere else and is not supposed to match today's date?

Reply:
It needs to be manually specified, so I went ahead and updated the date.